elm.json

```diff
@@ -2,6 +2,7 @@
     "type": "application",
     "source-directories": [
         "src",
+        "codegen",
```

Comment:
What is the codegen source directory for in this case?

Reply:
It's for all the bindings: all the modules whose name start with `Gen.`

I think we probably want to not `.gitignore` it actually, because `lamdera`

Reply:
Yeah, I don't think gitignoring it is a good idea. I tried running your branch locally and now I can't run the tests because I need to learn how to run codegen first.